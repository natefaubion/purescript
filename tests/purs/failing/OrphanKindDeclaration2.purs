-- @shouldFailWith OrphanKindDeclaration
module Main where

type Foo :: Type
data Foo = Foo Int
