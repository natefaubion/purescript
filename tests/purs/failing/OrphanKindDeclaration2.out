Error found:
in module [33mMain[0m
at tests/purs/failing/OrphanKindDeclaration2.purs:4:1 - 4:17 (line 4, column 1 - line 4, column 17)

  The kind declaration for [33mFoo[0m should be followed by its definition.


See https://github.com/purescript/documentation/blob/master/errors/OrphanKindDeclaration.md for more information,
or to contribute content related to this error.

